app/views/welcome/index.html.slim

```diff
@@ -8,7 +8,8 @@
     .row
       .col-sm-3.col-sm-offset-2
         .block
-          = mustdown t('.show-off.members_count'), members: members_count
```

Comment:
Before:

![Screenshot_2021-11-06_22-31-28](https://user-images.githubusercontent.com/1866809/140624438-36cadb2f-87c9-4f7e-8821-cc4e2f0f342f.png)

After

![Screenshot_2021-11-06_22-31-55](https://user-images.githubusercontent.com/1866809/140624443-cacbd31c-0e0b-4e1c-b270-2445ce34a6f6.png)

(it is the same)